pkgs/data/fonts/google-fonts/default.nix

```diff
@@ -18,10 +20,6 @@ stdenv.mkDerivation rec {
     chmod -x $dest/*.ttf
   '';
 
-  outputHashAlgo = "sha256";
-  outputHashMode = "recursive";
-  outputHash = "0q03gg0sh2mljlbmhamnxz28d13znh9dzca84p554s7pwg6z4wca";
```

Comment:
If the archives are generated with different timestamps, how do they have the same hash? I tried to set the timestamp to a constant one with `touch -t 0001010000`, but haven't had success. Regardless, the build is reproducible neither with `fetchurl` (I got a hash different from the original one) nor with `fetchFromGitHub` (@offlinehacker got a hash different from mine).